src/Twig/Extension/PageExtension.php

```diff
@@ -39,45 +39,24 @@
  */
 class PageExtension extends AbstractExtension implements InitRuntimeInterface
 {
-    /**
-     * @var CmsManagerSelectorInterface
-     */
-    private $cmsManagerSelector;
+    private CmsManagerSelectorInterface $cmsManagerSelector;
 
-    /**
-     * @var SiteSelectorInterface
-     */
-    private $siteSelector;
+    private SiteSelectorInterface $siteSelector;
 
     /**
      * @var array
      */
     private $resources;
 
-    /**
-     * @var Environment
-     */
-    private $environment;
+    private ?Environment $environment = null;
```

Comment:
is it nullable?

Reply:
Not initialized in the constructor but in
```
public function initRuntime(Environment $environment)
    {
        $this->environment = $environment;
    }
```